manifest.yml

```diff
@@ -439,12 +440,12 @@ instance_groups:
           # Monitoring auth
           prometheus-staging:
             <<: *client-template
-            scope: openid,email,profile
+            scope: openid,email,profile,prometheus.support,concourse.pages,concourse.admin
```

Comment:
I'm not familiar with this part of the ops-login manifest; can you tell me more about the additional scopes being granted?

Reply:
I wasn't either before McG showed me. Basically we're making fine grain access to Grafana right now. 

The new perm is prometheus.support, so named for a future ticket that will tackle prometheus itself. concourse.pages and concourse.admin are the overall perms currently given to platform folks and pages folks respectively. 

So the perms are really just a new scope for uaa and a way to control access to Grafana, rather than just any uaa user.

Reply:
Ok, those roles make sense. Just to confirm — Is this configuring OpsUAA with scopes related to Prometheus? E.g. is this the full list of scopes related to Prometheus that can be assigned to users?